### ISSUE TITLE
Unexpected token: class (PHP 5.5)

### ISSUE THREAD

**Issue by pkordylewski** [pdepend/pdepend#225]
Running latest stable pdepend:

```
./pdepend --version
PDepend 2.2.1
```

We get the following error:

```
[phpdepend] Following errors occurred:
[phpdepend] Unexpected token: class, line: 157, col: 21, file: /path/to/file
```

The code looks like this and is valid in PHP 5.5:

```
class someClass extends otherClass {
    public static function foobar() {
        if (static::class === self::class) {
            // some code
        }
    }
}
```

Cheers,
Patryk

**Comment by jdeniau** [comment]
:+1:

**Comment by manuelpichler** [comment]
Should be fixed with commit f59212704ac6dc7b502bb85dfd62e2469d051269
